packages/flutter_driver/lib/src/extension.dart

```diff
@@ -73,15 +75,16 @@ class _FlutterDriverExtension {
     });
 
     _commandDeserializers.addAll(<String, CommandDeserializerCallback>{
-      'get_health': (Map<String, dynamic> json) => new GetHealth.deserialize(json),
-      'get_render_tree': (Map<String, dynamic> json) => new GetRenderTree.deserialize(json),
-      'tap': (Map<String, dynamic> json) => new Tap.deserialize(json),
-      'get_text': (Map<String, dynamic> json) => new GetText.deserialize(json),
-      'scroll': (Map<String, dynamic> json) => new Scroll.deserialize(json),
-      'scrollIntoView': (Map<String, dynamic> json) => new ScrollIntoView.deserialize(json),
-      'setInputText': (Map<String, dynamic> json) => new SetInputText.deserialize(json),
-      'submitInputText': (Map<String, dynamic> json) => new SubmitInputText.deserialize(json),
-      'waitFor': (Map<String, dynamic> json) => new WaitFor.deserialize(json),
+      'get_health': (Map<String, String> params) => new GetHealth.deserialize(params),
+      'get_render_tree': (Map<String, String> params) => new GetRenderTree.deserialize(params),
+      'tap': (Map<String, String> params) => new Tap.deserialize(params),
+      'get_text': (Map<String, String> params) => new GetText.deserialize(params),
+      'set_frame_sync': (Map<String, String> params) => new SetFrameSync.deserialize(params),
```

Comment:
Not that I have better ideas, but "frame sync" might be confusing in that it might imply that you are attempting to sync with the beginning or end of a specific frame.